tests/gcloud-testing-core/src/main/java/com/google/cloud/runtime/jetty/util/HttpUrlUtil.java

```diff
@@ -87,7 +86,6 @@ public static HttpURLConnection openTo(URI uri) throws IOException {
     log.info("HttpUrlUtil.openTo(" + uri + ")");
     HttpURLConnection http = (HttpURLConnection) uri.toURL().openConnection();
     http.setRequestProperty("User-Agent", "jetty-runtime/gcloud-util-core");
-    http.setInstanceFollowRedirects(false);
```

Comment:
I still think that this test should not follow redirects, specially as we wish to test http vs https headers and there can be redirects from http to https.

Even if we do want to follow redirects, then I think it would be better to explicitly say so in the test, because it can be non obvious when trying to debug tests that redirection is happening.